js/product/product.js

```diff
@@ -20,16 +24,24 @@ const product = {
 
         axios.get(url, { params })
             .then(res => {
-                products = res.data
+                products = res.data.data
+                const pagination = res.data.pagination
+                this.currentPage = pagination._page
+                this.pageLimit = pagination._limit
+                this.totalPage = pagination._totalRows
+                const pageSize = this.handlePageSize(this.totalPage/this.pageLimit)
```

Comment:
this.totalPage/this.pageLimit nhớ Math.ceil
Ví dụ 1 trang max 10 sản phẩm. Mà có 101 sản phẩm thì nó phải là 11 trang

Reply:
mình có custom 1 hàm để xử lí rồi , quyên mất có cái hàm Math.ceil